README.md

```diff
@@ -109,6 +109,36 @@ You can run the application with this command: `air`.
 
 It will generate and reload thyra each time a file is modified.
 
+### Plugin installation
+
+Thanks to its modular architecture, it is possible to use third party plugins in Thyra.
+Plugins must be placed in a ```plugins``` folder next to the Thyra binary.
```

Comment:
Since in https://github.com/massalabs/thyra/issues/214 we moved the wallets from Thyra binary directory to `$HOME/.config/thyra/`, I think we should do the same for the plugins and so, store the `plugins` directory in `$HOME/.config/thyra/`.